build.gradle

```diff
@@ -1,12 +1,31 @@
+buildscript {
+    ext.ideaVersion = ideaVersion
+    /*
+     * for more complex conversion ideaVersion -> sinceIdeaBuild
+     * see https://github.com/rhdunn/xquery-intellij-plugin/blob/master/build.gradle#L1-L47
+     */
+    def since = ideaVersion =~ /IC-20([0-9][0-9])\.([1-3])(\.[0-9]+)?$/
```

Comment:
Why don't you use ideaVersion.substring(3, 6)

Reply:
need to also remove the '.' (IC-2019.3 -> 193)
Btw. this PR is about releasing a fix-version. I'll rebase it once #243 is merged.